config.toml

```
@@ -248,6 +248,115 @@ CacheCapacity = 1000
+		# would be a good set of keys in order to let you see accurately use of all
+		# endpoints and call out when there is failing traffic to any endpoint. Field
+		# names may come from any span in the trace.
+		# Eligible for live reload.
+		FieldList = ["request.method","response.status_code"]
+
+		# UseTraceLength will add the number of spans in the trace in to the dynamic
+		# sampler as part of the key. The number of spans is exact, so if there are
+		# normally small variations in trace length you may want to leave this off. If
+		# traces are consistent lengths and changes in trace length is a useful
+		# indicator of traces you'd like to see in Honeycomb, set this to true.
+		# Eligible for live reload.
+		UseTraceLength = true
+
+		# AddSampleRateKeyToTrace when this is set to true, the sampler will add a field
+		# to the root span of the trace containing the key used by the sampler to decide
+		# the sample rate. This can be helpful in understanding why the sampler is
+		# making certain decisions about sample rate and help you understand how to
+		# better choose the sample rate key (aka the FieldList setting above) to use.
+		AddSampleRateKeyToTrace = true
+
+		# AddSampleRateKeyToTraceField is the name of the field the sampler will use
+		# when adding the sample rate key to the trace. This setting is only used when
+		# AddSampleRateKeyToTrace is true.
+		AddSampleRateKeyToTraceField = "meta.samproxy.dynsampler_key"
+
+		# AdjustmentInterval defines how often (in seconds) we adjust the moving average from
+		# recent observations. Default 15s
+		# Eligible for live reload.
+		AdjustmentInterval = 15
+
+		# Weight is a value between (0, 1) indicating the weighting factor used to adjust
+		# the EMA. With larger values, newer data will influence the average more, and older
+		# values will be factored out more quickly.  In mathematical literature concerning EMA,
+		# this is referred to as the `alpha` constant.
+		# Default is 0.5
+		# Eligible for live reload.
+		Weight = 0.5
+
+		# MaxKeys, if greater than 0, limits the number of distinct keys tracked in EMA.
+		# Once MaxKeys is reached, new keys will not be included in the sample rate map, but
+		# existing keys will continue to be be counted. You can use this to keep the sample rate
+		# map size under control.
+		# Eligible for live reload
+		MaxKeys = 0
+
+		# AgeOutValue indicates the threshold for removing keys from the EMA. The EMA of any key will approach 0
+		# if it is not repeatedly observed, but will never truly reach it, so we have to decide what constitutes "zero".
+		# Keys with averages below this threshold will be removed from the EMA. Default is the same as Weight, as this prevents
+		# a key with the smallest integer value (1) from being aged out immediately. This value should generally be <= Weight,
+		# unless you have very specific reasons to set it higher.
+		# Eligible for live reload
```

This is the only paragraph in this block that is not indented to 80char
```suggestion
		# AgeOutValue indicates the threshold for removing keys from the EMA. The EMA of any key
		# will approach 0 if it is not repeatedly observed, but will never truly reach it, so we have to
		# decide what constitutes "zero". Keys with averages below this threshold will be removed
		# from the EMA. Default is the same as Weight, as this prevents a key with the smallest
		# integer value (1) from being aged out immediately. This value should generally be <= Weight,
		# unless you have very specific reasons to set it higher.
		# Eligible for live reload
```